libs/stats/README.md

```diff
@@ -13,21 +12,84 @@ pip install --extra-index-url="https://packages.dea.ga.gov.au" odc-stats
 
 Usage
 -----
+Stats offers a set of tools to generate clear pixel count and geometric median on Sentinel-2 and Landsat satellite images.
+### Steps to run stats
 
-### Prepare tasks for processing:
+### 1- Save tasks
 
 From a machine that has access to your database, run:
 
 ```
-odc-stats save-tasks --year YYYY --grid  grid-name  --temporal-range temporal-range product-name
+odc-stats save-tasks --year YYYY --grid  grid-name  --temporal-range range product-name
 ```
-This will generate a cache file that contains tasks.  For more options, check the help.  
-### Publish tasks 
+This will generate a csv file containing the list of tasks to be processed, a cache file that will be used as an input to the stats cli, and geojson files.  
 
+
+
+### 2- Publish tasks 
```

Comment:
This section should be AFTER "run section". Let's first describe how to test configuration/code, and only then focus on large scale processing.